src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -2157,6 +2190,7 @@ public function toDataSource($mixins = [])
         $data['photo'] = $this->getProfilePhoto() === null ?
             Config::$default_person_uri : $this->getProfilePhoto()->getURL();
         $data['bio'] = $this->getBio();
+        $data['radio-time'] = $this->getRadioTime();
```

Comment:
This isn't a valid JS identifier, so accessing it from `JSON.parse()` is pain. Consider `radioTime` or `radio_time`.